### PR TITLE
test old jinja2 and 3.x

### PR DESCRIPTION
**Pull request by mmerickel**

### CODE REVIEW

File: src/pyramid_jinja2/filters.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
-from jinja2 import pass_context
 from pyramid.threadlocal import get_current_request
 from pyramid.url import resource_url, route_path, route_url, static_path, static_url
 
+try:
+    from jinja2 import pass_context
+except ImportError:
+    # jinja2 < 3.0 fallback
+    from jinja2 import contextfilter as pass_context
```
Comment (stevepiercy) [inline src/pyramid_jinja2/filters.py:8]: I probably don't understand the coverage report, but in the "Validate coverage" CI run, [lines 6-8 were missed](https://github.com/Pylons/pyramid_jinja2/runs/5548626368?check_suite_focus=true#step:5:40), but the run reports it at 100% coverage. I understand why they were missed, but do not understand why the combined report seems to ignore it.

Comment (mmerickel) [reply src/pyramid_jinja2/filters.py:8]: just look at the coverage step in the ci config (that I changed as part of this pr)

Comment (stevepiercy) [reply src/pyramid_jinja2/filters.py:8]: Right, [that](https://github.com/Pylons/pyramid_jinja2/pull/164/files#diff-03609cb60b0c6e92fb771eb8787d6722b8c31ca4c03eabc788e147acd8c6fb43R87) explains why they were missed, but does the combination of the two envs preceding `coverage` mean the union of them?

Comment (mmerickel) [reply src/pyramid_jinja2/filters.py:8]: yes

**Pull request merged**